tools/proofers/my_projects.php

```diff
@@ -83,6 +83,18 @@
 
 echo "<h1>" . get_usertext($page_header) . "</h1>";
 
+$can_view_post_processing = in_array("PP", $allowed_stages) or in_array("PPV", $allowed_stages);
+$proof_heading = _("Proofreading & Formatting Projects");
+$pool_heading = _("Post-Processing Projects");
+if ($can_view_post_processing)
+{
+    echo "<div style='padding: .5em 0'>";
```

Comment:
We should make this a first-class style and add it to the main style sheet in `layout.less`. I would also like to see this element defined in a way that we don't have to specify the vertical bar, something like:
```
<div class='quicklinks'>
<a href=...>Link1</a>
<a href=...>Link2</a>
</div>
```
That should be doable with the right CSS. I don't really care too much if we need to define this as a `<ul>` instead of a `<div>`.

We should add this to the `styles/style_demo.php` page as well.

Reply:
Here it is on the demo page:
![image](https://user-images.githubusercontent.com/1526781/87494518-dac1c900-c614-11ea-8408-04886195bb6c.png)